lib/src/Uncertainty/Distribution/openturns/ComposedDistribution.hxx

```diff
@@ -213,6 +213,9 @@ private:
   /** Compute the covariance of the distribution */
   void computeCovariance() const override;
 
+  /* String formatter with right padding */
```

Comment:
Still there?

Reply:
Oups. Sorry for that. Fixed at 84c7d7740